hw/net/rocker/rocker_hw.h

```diff
@@ -165,6 +167,14 @@ enum {
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
 
+    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
```

Comment:
Are these STATS updates from David Ahern?  I know he's been patiently waiting to get some rocker device stats patches in.  I should just pull them in and update scottfeldman/qemu-rocker.